app/jobs/repository_audit_job.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class RepositoryAuditJob < ApplicationJob
+  def perform(user: User.system_user)
+    packages = Package.stored
+    audit = Audit.create(user: user, packages: packages.count)
+    packages.each do |package|
+      AuditFixityCheckJob.perform_now(package: package, user: user, audit: audit)
```

Comment:
Whoops! We can't `perform_now` on the individual items. We just want to initiate the audit, scheduling the individual packages checks (`peform_later`), and then return the audit.